regression-test/suites/nereids_p0/show/test_show_sync_job_command.groovy

```diff
@@ -0,0 +1,56 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_show_sync_job_command", "query,sync_job") {
+    try {
+        sql """CREATE DATABASE IF NOT EXISTS test_db;"""
```

Comment:
pls dont use common name like test_db. it might collide with other testcase and cause random failures.
pls use like test_show_sync_job_command_db.

Reply:
done - pls review